src/stores/canvas.ts

```diff
@@ -41,6 +45,16 @@ const useStoreCanvas = defineStore({
       })
       await promise
     },
+    async resetCanvas(stageParentDiv: HTMLDivElement) {
```

Comment:
```suggestion
    resetCanvas() {
```
リセットしたキャンバスをfirebaseに保存しなくなったので、async関数にする必要はなくなったと思います。

Reply:
見逃してました。